OrbitQt/orbitmainwindow.cpp

```diff
@@ -71,8 +71,14 @@ OrbitMainWindow::OrbitMainWindow(QApplication* a_App,
   ui->HomeHorizontalSplitter->setSizes(sizes);
   ui->splitter_2->setSizes(sizes);
 
-  GOrbitApp->AddCaptureStartedCallback(
-      [this] { ui->HomeTab->setDisabled(true); });
+  GOrbitApp->AddCaptureStartedCallback([this] {
+    ui->actionOpen_Capture->setDisabled(true);
```

Comment:
What's the current appearance this will cause? Grayed out?

Reply:
Unfortunately, not exactly, this is probably related to the fact that we don't yet have a proper dark-theme style sheet.

![Untitled](https://user-images.githubusercontent.com/58685940/84756587-45a2c480-afc3-11ea-8de8-a0d7246e2499.png)

Reply:
Ok, not a big concern for me. Disabling is the correct thing to do here, we just need to fix our style sheet (separately from this PR).

Reply:
Is that alright for now?

Reply:
Our comments just crossed, yes, this is fine IMO.